### ISSUE TITLE
Migrating Gazebo 7 tags to Xenial base image

### ISSUE THREAD

**Issue by ruffsl** [osrf/docker_images#92]
Back when I added the Dockerfiles for Gazebo v7, I listed Trusty as the base image, at its was the latest LTS listed as supported from the `Release Schedule and Roadmap` on the official gazebo site:
http://gazebosim.org/#collapseVersion7_1

![image](https://user-images.githubusercontent.com/2293573/31240761-93668b9a-a9b6-11e7-9814-4e88d320f56a.png)

@mikaelarguedas informed me that it has been ported to Xenial, and is listed under the package index, ~line 20: http://packages.osrfoundation.org/gazebo/ubuntu-stable/dists/xenial/main/binary-amd64/Packages

Plus, given the EOL date for Gazebo v7 matches Xenial, so this is a good case to migrate the v7 tags to build `FROM` Trusty to Xenial.

Lucky, Gazebo does not define OS level specific tags, so we don't need to support more tags than needed.

**Comment by ruffsl** [comment]
Cross posted to gazebosim community [here](https://community.gazebosim.org/t/migrating-gazebo-7-docker-tags-to-xenial-base-image/27).

**Comment by mikaelarguedas** [comment]
Yeah my [initial remark](https://github.com/osrf/docker_images/pull/88#discussion_r142828296) is that gazebo7 is an upstream ubuntu package as well but only in xenial and later (see [packages.ubuntu.com](https://packages.ubuntu.com/search?keywords=gazebo&searchon=names&suite=all&section=all)).
Given that gazebo 7 has the same lifetime as xenial and kinetic (the ROS distro targeting gazebo7) I think it would make more sense to base the gazebo7 image on xenial
